flask_simplelogin/__init__.py

```diff
@@ -126,26 +129,40 @@ def wrap(*args, **kwargs):
     return decorator
 
 
+class Message():
+    def __init__(self, text, category="primary", enabled=True):
+        self.text = text
+        self.category = category
+        self.enabled = True
+
+    def flash(self):
+        if self.text and self.enabled:
+            flash(self.text, self.category)
+
+
 class SimpleLogin(object):
     """Simple Flask Login"""
 
     messages = {
-        'login_success': 'login success!',
-        'login_failure': 'invalid credentials',
-        'is_logged_in': 'already logged in',
-        'logout': 'Logged out!',
-        'login_required': 'You need to login first',
-        'access_denied': 'Access Denied',
-        'auth_error': 'Authentication Error: {0}'
+        'login_success': Message('login success!', category='success'),
+        'login_failure': Message('invalid credentials', category='danger'),
+        'is_logged_in': Message('already logged in', category='primary'),
```

Comment:
Can we omit this `category='primary'`?

Reply:
Yes, `category='primary'` can be omitted. It may be worth leaving in for readability, but it's not needed.

Reply:
What really bugs me is the apparent inconsistency: why do we (arguably) care for readability in `is_logged_in` and `logout`, but not in `access_denied` or `auth_error`?

I get that they are not flashed, but for consistency, I would omit all of them (it sounds better than being explicit in all of them, including cases in which it's meaningless)…

Alternatively we can use something like `category=None` in places where this argument is not used… what do you think?